README.rst

```diff
@@ -43,13 +43,31 @@ libraries in the ``pep0`` directory.
 Checking PEP formatting and rendering
 =====================================
 
-Do not commit changes with bad formatting.  To check the formatting of
-a PEP, use the Makefile.  In particular, to generate HTML for PEP 9999,
-your source code should be in ``pep-9999.rst`` and the HTML will be
-generated to ``pep-9999.html`` by the command ``make pep-9999.html``.
-The default Make target generates HTML for all PEPs.
-
-If you don't have Make, use the ``pep2html.py`` script directly.
+Avoid committing changes with reStructuredText syntax errors that cause PEP
```

Comment:
I prefer the strict admonition of the prior text ("Do not commit ...") -- it is much clearer than "avoid"

Reply:
I changed it to "Please don't" to still carry the same weight, but be a little less harsh. I also fixed some editing typos I spotted in my changes.

Reply:
"please" is almost always an unneeded word in documents like this -- as Hugo said readers tend to scan, so being nice is somewhat wasted!

A

Reply:
It may be strictly semantically unnecessary, but it just sounds unnecessarily harsh and intimidating to be telling new contributors "Don't commit changes with syntax errors", like telling them "Don't make mistakes!"